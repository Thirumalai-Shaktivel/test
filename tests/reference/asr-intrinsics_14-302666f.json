{
    "basename": "asr-intrinsics_14-302666f",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_14.f90",
    "infile_hash": "8055dcd4ac420a18e32d888002cf97b13bbe9009820c508a4a91b174",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-intrinsics_14-302666f.stdout",
    "stdout_hash": "411a8581940aa8a5b0b9005b842f696b55699f0187222693e41e8afa",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}