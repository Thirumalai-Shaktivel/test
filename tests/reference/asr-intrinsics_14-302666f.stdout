(TranslationUnit (SymbolTable 1 {intrinsics_14: (Program (SymbolTable 2 {selected_int_kind: (ExternalSymbol 2 selected_int_kind 4 selected_int_kind lfortran_intrinsic_kind [] selected_int_kind Private), selected_real_kind: (ExternalSymbol 2 selected_real_kind 4 selected_real_kind lfortran_intrinsic_kind [] selected_real_kind Private)}) intrinsics_14 [] [(Print () [(FunctionCall 2 selected_int_kind () [(ConstantInteger 5 (Integer 4 []))] [] (Integer 4 []) (ConstantInteger 4 (Integer 4 [])) ())]) (Print () [(FunctionCall 2 selected_int_kind () [(ConstantInteger 5 (Integer 4 []))] [] (Integer 4 []) (ConstantInteger 4 (Integer 4 [])) ())]) (Print () [(FunctionCall 2 selected_int_kind () [(ConstantInteger 5 (Integer 8 []))] [] (Integer 4 []) (ConstantInteger 4 (Integer 8 [])) ())]) (Print () [(FunctionCall 2 selected_int_kind () [(ConstantInteger 8 (Integer 4 []))] [] (Integer 4 []) (ConstantInteger 4 (Integer 4 [])) ())]) (Print () [(FunctionCall 2 selected_int_kind () [(ConstantInteger 10 (Integer 4 []))] [] (Integer 4 []) (ConstantInteger 8 (Integer 4 [])) ())]) (Print () [(FunctionCall 2 selected_real_kind () [(ConstantInteger 5 (Integer 4 []))] [] (Integer 4 []) (ConstantInteger 4 (Integer 4 [])) ())]) (Print () [(FunctionCall 2 selected_real_kind () [(ConstantInteger 5 (Integer 4 []))] [] (Integer 4 []) (ConstantInteger 4 (Integer 4 [])) ())]) (Print () [(FunctionCall 2 selected_real_kind () [(ConstantInteger 5 (Integer 8 []))] [] (Integer 4 []) (ConstantInteger 4 (Integer 8 [])) ())]) (Print () [(FunctionCall 2 selected_real_kind () [(ConstantInteger 8 (Integer 4 []))] [] (Integer 4 []) (ConstantInteger 8 (Integer 4 [])) ())])]), lfortran_intrinsic_kind: (IntrinsicModule lfortran_intrinsic_kind)}) [])
